redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -335,5 +342,297 @@ inline void
+                                }
+                                else if (property.first == "Severity")
+                                {
+                                    const std::string* severity =
+                                        std::get_if<std::string>(
+                                            &property.second);
+                                    if (severity == nullptr)
+                                    {
+                                        BMCWEB_LOG_ERROR
+                                            << "Failed to get the Severity "
+                                            << "from object: "
+                                            << hwStatusEventObj;
+                                        messages::internalError(aResp->res);
+                                        break;
+                                    }
+
+                                    if (*severity ==
+                                        "xyz.openbmc_project.Logging.Event."
+                                        "SeverityLevel.Critical")
+                                    {
+                                        condition["Severity"] = "Critical";
+                                    }
+                                    else if ((*severity ==
+                                              "xyz.openbmc_project.Logging."
+                                              "Event.SeverityLevel.Warning") ||
+                                             (*severity ==
+                                              "xyz.openbmc_project.Logging."
+                                              "Event.SeverityLevel.Unknown"))
+                                    {
+                                        condition["Severity"] = "Warning";
+                                    }
+                                    else if (*severity ==
+                                             "xyz.openbmc_project.Logging."
+                                             "Event.SeverityLevel.Ok")
+                                    {
+                                        condition["Severity"] = "OK";
+                                    }
+                                    else
+                                    {
+                                        BMCWEB_LOG_ERROR
+                                            << "Unsupported Severity[ "
+                                            << *severity << "] from object: "
+                                            << hwStatusEventObj;
+                                        messages::internalError(aResp->res);
+                                        break;
+                                    }
+                                }
+                            }
+                        },
+                        objType[0].first, hwStatusEventObj,
+                        "org.freedesktop.DBus.Properties", "GetAll", "");
+                },
+                "xyz.openbmc_project.ObjectMapper",
```

Comment:
Why 2 mapper calls here?

Reply:
Assuming this is clarified so marking as resolved.